### PR TITLE
Graph Edge Import

### PR DESCRIPTION
**Pull request by diana-macrometa**
Final PR for the GDN Collection Types import.

- Added instructions for adding a Graph Edge collection
- Added instructions for adding edges to a Graph Edge collection
- A few other minor fixes

### CODE REVIEW

File: docs/collections/graphs/add-edges.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to add edges to a Graph Edge collection.
+
+## Add an Edge with the Console
+
+Follow these instructions to add documents to an existing document collection using the GDN console web UI.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]:
```suggestion
To add edge documents to a collection:
```

File: docs/collections/graphs/add-edges.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+1. Log in to your Macrometa account.
+1. Click **COLLECTIONS**.
+2. In the collection list, click the name of the edge collection to which you want to add an edge. If you aren't sure which collections are Graph Edge collections, then you can click **Edge** at the top of the page to see just Graph Edge collections.
+3. Click **New Document**.
+4. Enter information in the fields.
+   - **_from -** Document _id from which the relationship originates.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]: Should we use code tag for these descriptions, i.e.:

```suggestion
   - **_from -** Document `_id` from which the relationship originates.
```

File: docs/collections/graphs/add-edges.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+   - **_to -** Document _id to which the relationship is defined.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]: Same Q as above:

```suggestion
   - **_to -** Document `_id` to which the relationship is defined.
```

File: docs/collections/graphs/add-edges.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+   - **_key -** Optional. If left blank, then Macrometa automatically generates a key.
+5. Click **Create**.
+
+## Add Edges from a File
+
+Follow these instructions to add documents to an existing document collection from a JSON or CSV file.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]:
```suggestion
To add edge documents to a collection from a JSON or CSV file:
```

File: docs/collections/graphs/add-edges.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+![Import a Document](/img/collections/import-docs.png)
+
+1. Log in to your Macrometa account.
+1. Click **COLLECTIONS**.
+1. In the collection list, click the name of the edge collection to which you want to add an edge. If you aren't sure which collections are Graph Edge collections, then you can click **Edge** at the top of the page to see just Graph Edge collections.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md:33]:
```suggestion
1. In the collection list, click the name of the edge collection to which you want to add an edge. 

    Click **Edge** at the top of the page to filter everything but edge collections.
```

@diana-macrometa Don't think we've discussed this, but for secondary statements/supporting statements in steps, do we want to have them on the same line or a separate line nested beneath the step (as I did above)?  Also, not sure I formatted it 100% properly

File: docs/collections/graphs/add-edges.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+1. Click the import icon, which is a down arrow pointing to a file box.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md:34]:
```suggestion
1. Click the import icon (down arrow pointing to a file box).
```

@diana-macrometa Any opinions on parentheses versus simply having a dependent clause?  This way is slightly less wordy, but the parens clutter things up a bit.

Either way, we can probably delete this explanation once we have screenshots.

File: docs/collections/graphs/add-edges.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    client.insert_document(collection_name='employees', document=docs)
+```
+
+  </TabItem>
+  <TabItem value="js" label="Javascript">
+
+```js
+    const jsc8 = require("jsc8");
+
+    // Crete a authenticated instance with Token / Apikey
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]:
```suggestion
    // Create an authenticated instance with a token or API key.
```

File: docs/collections/graphs/add-edges.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    // const client = new jsc8({url: "https://gdn.paas.macrometa.io", token: "XXXX", fabricName: '_system'});
+    // const client = new jsc8({url: "https://gdn.paas.macrometa.io", apiKey: "XXXX", fabricName: '_system'});
+    // await console.log("Authentication done!!...");
+
+    // Or use Email and Password to Authenticate client instance
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/add-edges.md]:
```suggestion
    // Or use email and password to authenticate client instance.
```

File: docs/collections/graphs/create-graph-edge.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+---
+title: Create Graph Edge
+sidebar_position: 20
+---
+
+The Graph Edge collection is a form of document collection. It is used define relationships between two documents by using the **_from** and **_to** system attributes and the document **_id** system attribute.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/create-graph-edge.md:6]: @diana-macrometa So far we've been using "edge" as an improper noun, do we want to reconsider this and maybe use "Edge" as a proper noun?

File: docs/collections/graphs/create-graph-edge.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+## Create a Document Store Collection with the Console
+
+Follow these instructions to create a new collection using the GDN console web UI.
```
Comment (jamesm-macrometa) [inline docs/collections/graphs/create-graph-edge.md:10]:
```suggestion
To create a collection:
```

**Review by jamesm-macrometa — APPROVED** [review]
Minor suggestions, looks good!

File: docs/collections/graphs/add-edges.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/collections/graphs/add-edges.md]: They actually call them "edges." It's wacky.

File: docs/collections/graphs/create-graph-edge.md (hunk 9 above)
Comment (diana-macrometa) [reply docs/collections/graphs/create-graph-edge.md:6]: No, I'm using "edge" as they use it throughout the section. It's generic in this usage.

**Pull request merged**